modules/python/python-dest.c

```diff
@@ -208,8 +208,9 @@ _py_invoke_open(PythonDestDriver *self)
     {
       if (ret == Py_None)
         {
-          msg_warning_once("Since " VERSION_3_25 ", the return value of open method in python destination "
-                           "is used as success/failure indicator. Please use return True or return False explicitly",
+          msg_warning_once("python-dest: Since " VERSION_3_25 ", the return value of the open() method "
+                           "is used as success/failure indicator. Please use return True or return False explicitly "
+                           "instead of returning None",
```

Comment:
This warning message is also shown when the user code does not return anything.

Reply:
if the user code does not have a return statement, that is the same as returning None in my book. but you are right,
users may be inexperienced Python coders, so omitting that extra bit of information.